.ruby-version

```diff
@@ -0,0 +1 @@
+3.4.2
```

Comment:
So this gem supports any ruby version >= 2.7.0. Is 3.4.1 too strict given it is an open source gem?

> s.required_ruby_version = ">= 2.7.0"